driving_log_replayer/scripts/perception_2d_evaluator_node.py

```diff
@@ -303,7 +304,7 @@ def list_dynamic_object_2d_from_ros_msg(
                 perception_object.object.classification
             )
             label = self.__evaluator.evaluator_config.label_converter.convert_label(
-                label=get_label(most_probable_classification)
+                name=get_label(most_probable_classification), attributes=[]
```

Comment:
[NITS] You don't have to specify `attributes` for estimated objects

```suggestion
                name=get_label(most_probable_classification)
```